homework_04/13_martin_georgiev/circles_hw.py

```diff
@@ -1,27 +1,43 @@
+'''
+file with circles and util for determining position and colisions
+'''
+# pylint: disable=C0103
+# pylint: disable=R0903:
```

Comment:
I am not sure for what you have had C0103 (I really do not see any problems for this, you can check this, if you want: https://pycodequ.al/docs/pylint-messages/c0103-invalid-name.html). For R0903: a simple solution is to add properties or override equal methods. I just mention it, this way is very good too.